src/emcee/ensemble.py

```diff
@@ -15,11 +15,11 @@
 
 __all__ = ["EnsembleSampler", "walkers_independent"]
 
-try:
-    from collections.abc import Iterable
-except ImportError:
-    # for py2.7, will be an Exception in 3.8
-    from collections import Iterable
+from collections.abc import Iterable, Sequence
+
+ParameterNamesT = Union[
+    Sequence[str], Dict[str, Union[slice, int, Sequence[int]]]
+]
```

Comment:
This fails on import at least on Python 3.7 and 3.8. Perhaps it makes more sense to import them from `typing`? I think that would work on both of those platforms?

Reply:
Thanks for catching that - 2da5cd2.